code/modules/client/client_procs.dm

```diff
@@ -59,7 +59,10 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 			var/msg = "Your previous action was ignored because you've done too many in a minute."
 			if (minute != topiclimiter[ADMINSWARNED_AT]) //only one admin message per-minute. (if they spam the admins can just boot/ban them)
 				topiclimiter[ADMINSWARNED_AT] = minute
-				msg += " Administrators have been informed."
```

Comment:
You completely got rid of the `Administrators have been informed.` part of the message?

Reply:
Yes, Jacquerel said it's not useful information and just serves to alarm players

Reply:
oh I see, fair enough ig